libs/api/ebpf_api.cpp

```diff
@@ -4655,8 +4665,7 @@ ebpf_ring_buffer_map_subscribe(
 CATCH_NO_MEMORY_EBPF_RESULT
 
 _Must_inspect_result_ ebpf_result_t
-ebpf_ring_buffer_map_write(fd_t map_fd, _In_reads_bytes_(data_length) const void* data, size_t data_length)
-    NO_EXCEPT_TRY
+ebpf_map_write(fd_t map_fd, _In_reads_bytes_(data_length) const void* data, size_t data_length) NO_EXCEPT_TRY
```

Comment:
```suggestion
static
_map_write(fd_t map_fd, _In_reads_bytes_(data_length) const void* data, size_t data_length) NO_EXCEPT_TRY
```
If this function is just local to this file, make it static. (but see my other comment)